module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -473,37 +366,164 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::goalieWalk(const Field& field, const Ball& ball) {
+
+        // make a motionCommand to emit
         std::unique_ptr<MotionCommand> motionCommand;
 
+        // time since the ball has been seen, in microseconds, current clock - time the ball was last measured
         float timeSinceBallSeen =
             std::chrono::duration_cast<std::chrono::microseconds>(NUClear::clock::now() - ballLastMeasured).count()
```

Comment:
```suggestion
            std::chrono::duration_cast<std::chrono::duration<float>>(NUClear::clock::now() - ballLastMeasured).count()
```

usually it would be something like `std::chrono::duration<float, std::milli>` but the default is `seconds` and there is no way to explicitly set it.

Reply:
trust lead clocking Kevin

Reply:
Done.